Samples/CameraSettingsExternalSettingsApp/DefaultControlHelper/DefaultControlHelper.h

```diff
@@ -15,32 +15,27 @@ namespace winrt::DefaultControlHelper::implementation
     struct DefaultControlManager : DefaultControlManagerT<DefaultControlManager>
     {
         DefaultControlManager(winrt::hstring cameraSymbolicLink);
+        ~DefaultControlManager();
 
         winrt::DefaultControlHelper::DefaultController CreateController(DefaultControllerType type, const uint32_t id);
 
-        wil::com_ptr_t<IMFCameraControlDefaultsCollection> GetCollection() { return m_controlDefaultsCollection; }
+        wil::com_ptr_t<IMFCameraControlDefaultsCollection> GetDefaultCollection();

```

Comment:
>_t [](http://example.com/codeflow?start=20&length=2)

nit: As you are editing some of these to `wil::com_ptr` can you do it for all of them?